src/run.py

```diff
@@ -1,58 +1,68 @@
 #add 2 numbers
 def add(x, y):
-    return x + y
+  return x + y
+
 
 #subtract 2 numbers
 def subtract(x, y):
-    return x - y
+  return x - y
+
 
 #multiply 2 numbers
 def multiply(x, y):
-    return x * y
+  return x * y
```

Comment:
Function comments should follow some kind of standard. The one I use looks like this:

```py
def multiply(x: float, y: float) -> float:
"""
Multiplies two numbers together

Parameters
----------
x : float
    first number to multiply

y : float
    second number to multiply

Returns
-------
float
    `x` multiplied by `y`. Mathematically equivalent to `x * y`
"""
return x * y
```

Also notice the `x : float` stuff in function parameters. This is a new feature in python 3.8 that allows to hint what types the functions expect to be passed in. The `-> float` at the end is what the function is expected to return.